packages/venia-ui/lib/components/Pagination/navButton.css

```diff
@@ -1,10 +1,18 @@
-.buttonArrow {
+.root {
     composes: button from './tile.css';
     display: grid;
     padding-bottom: 1px;
 }
 
-.buttonInactive {
-    composes: buttonArrow;
-    color: #999;
+.root_disabled {
+    composes: root;
```

Comment:
This class seems a bit redundant. Should this instead just be, or did you intend to add additional styles?
```css
.root:disabled {
  pointer-events: none;
}
```

Reply:
Fair point. I had included it as an optional classname to mirror the one for the icon, but the pseudo is probably sufficient for the root itself.

Reply:
While I was fixing this, I noticed a few more bits of weird CSS in the pagination styles. These have been fixed. Please check those out too. @tjwiebell